app/logic/events.py

```diff
@@ -446,3 +446,10 @@ def getTomorrowsEvents():
     tomorrowDate = date.today() + timedelta(days=1)
     events = list(Event.select().where(Event.startDate==tomorrowDate))
     return events
+def sortEvents(EventList):
```

Comment:
I would reverse this sorting mechanism, it doesn't make much sense to interrupt the flow of the events list by changing the way are sorted. I would recommend having the newer past events towards the top and older ones further down like how active ones are sorted.